### PR TITLE
Update dispute notification

### LINKED ISSUE #3781: notification card, markets in dispute
notification-state.ts

`selectMarketsInDispute` selector needs to find all markets in dispute where:
 * user is creator or 
 * has position or 
 * has disputed/reported

user's reporting/disputing history is located on state
`state.loginAccount.reporting`

Tasks:
 * Have the user click on the button and open reporting/disputing form with no outcome selected.

 * Timer is based on Markets dispute window endTime.

notification card just as reference, style should be updated as part of color changes and condensing.
image

<https://www.figma.com/file/bZh7tOiLm6Ls9Hl4RL6qMC5X/Augur-Redesign?node-id=7272%3A6488>

### PR DESCRIPTION
**Pull request by adrake33**
Closes #3781

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌈

 This link will always point to this PR's latest changes: https://peek.run/4kgh577

File: packages/augur-ui/src/modules/notifications/selectors/notification-state.ts [hunk 1]
```diff
@@ -94,17 +94,22 @@ export const selectMarketsInDispute = createSelector(
   selectAccountPositionsState,
   selectLoginAccountAddress,
   (markets, positions, address) => {
+    const state = store.getState();
+    const disputedMarkets = state
+      .loginAccount.reporting.disputing.contracts.map(obj => obj.marketId);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/notifications/selectors/notification-state.ts]: either one of these can be null. 
state.loginAccount.reporting.reporting
state.loginAccount.reporting.disputing

also, I don't see markets where user is creator.

**Review by bthaile — CHANGES_REQUESTED** [review]
need null checks and get markets where user created markets for dispute notification

**Comment by adrake33** [comment]
> need null checks and get markets where user created markets for dispute notification

Yeah, good point. I added checks to see if those are null. The check for whether the user is the creator is in the filter below: ```(market.reportingState ===
              MarketReportingState.CrowdsourcingDispute &&
              (market.author === address ||
                positionsMarkets.indexOf(market.id) > -1)```

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**